### DISCUSSION TITLE
Impact of Transitioning to KRE 3.1

### DISCUSSION THREAD

**Discussion by samdowd** [kinfoundation/kin-rfcs#120] [_ Archive - KRE 3.1 Discussion]
Great work on this, I like the spirit of these changes. However, the "Impact" section looks pretty concerning. The top 3 apps would be receiving 249 of the 250 million Kin (over 99%). That seems contrary to the stated objectives of the proposal. App 6 for example looks to have a reasonable economy but would lose ~90% of their KRE reward primarily to the apps at the top. I like that new apps are getting 50,000 Kin to get started, but if 2000 active users only adds another 50,000 Kin, there's still going to be an impassable valley between getting your first few users and getting your first 100,000.

My guess is a lot of this has to do with the fact that only 3 apps in the ecosystem are consistently doing 1000 Kin spends? But without more information, it's hard to tell how much of this imbalance will be fixed by apps adapting to that new requirement and how much won't. I understand a calculator is coming, but in the meantime, is it possible to share the numbers used to arrive at the KRE rewards in the Impact section? That would help me see in action how each "track" and the new spend guidelines are contributing to the end result. Clearly something is kneecapping apps 5-10, but it's hard to see what it is from that table.

If this impact is the final impact, I'd be very disappointed to see all but 3 apps lose all incentive to use Kin. But I'm hopeful that the impact we see here is simply an adjustment period and a reflection of just how good the spends in the top 3 apps are. I'd just like to see a little more data to see what the other apps would have to adjust to become competitive again.

Side note, can someone confirm how much of the new KRE that table takes into account? I see it's still using the old daily payout. Is it using the old or new guidelines for active users? Or are those only active users with 3 separate 1000 Kin transactions? Looking at app 1 at least, it looks like they have the same number of AU they had in KRE 3.0, but maybe app 1 only has >1000 Kin spends already.

**Comment by kin1076** [comment]
I agree that it looks ominous but I think apps will adapt and it forces them to become more creative in their spends. I think this might cause some apps to bail though but hopefully not.

**Reply by samdowd** [reply]
I think you're right. And a lot of the apps in that table with median spends of 2 or 3 Kin make sense to have their rewards cut down a lot unless they adapt. I'd just like to see some more of the variables that go into the calculations so I can see what's causing the drop on some of the apps that don't look that bad.

I'm also confused how an app can have a median spend of 3 if spends under 1000 aren't counted, so I think I must be understanding that whole table incorrectly.

**Reply by showered** [reply]
The table appears to include spends of 999 Kin and below. Removing these spends – I'm pretty sure – would make a huge difference to the MAS figures. It would also mean that Median spend would *necessarily* be above 1,000.

I think we all know the rules now, so it doesn't matter too much, but nevertheless it would be quite nice to see some illustrative figures based on the 1,000 minimum spend.

**Reply by skyf0xx** [reply]
@samdowd You make a very good point. These figures show the earnings before the Monopoly Clause is applied for apps. Showing them with the clause applied obfusicates the real effect of AUB, Spends and Balances. I will add some clarification to the proposal.

The [Monopoly Clause](https://github.com/kinecosystem/rewards-engine/blob/b63e38c3fd0eb110f58c85340bd561ee8593af12/current-KRE.md#monopoly-clause) states:
>No single developer will receive more than 66.67% of the KRE payout for a single day and any developer that would have received more than 50% of the payout will have their payout adjusted. No two developers will receive more than 90% of the KRE's payout for a single day. In both cases, residual payouts will be redistributed proportionally to other developers.

This will continue to apply, in addition to curving the rewards as proposed.

**Reply by samdowd** [reply]
Thanks, @hitwill! That makes sense. Can you clarify the following:
- Is the KRE 3.0 number it's comparing to pre- or post-Monopoly Clause?
- Why are the values in the table calculated including <1000 Kin spends if those no longer count?

**Reply by skyf0xx** [reply]
@samdowd 
question 1: The numbers shown in the table are *pre-monopoly* - we want the developers to see the actual effect of each factor. However after this, it will still be evened out by the monopoly clause
question 2: A good point - the fact is, if we showed only apps who currently have > 1,000 spends, several apps would be missing from the table. However, the calculation method still remains true. On launch, though, only > 1,000 spends will be considered.
